runtime/metricsview/query.go

```diff
@@ -186,6 +187,7 @@ type Condition struct {
 }
 
 type Subquery struct {
+	RawSQL    string      `mapstructure:"raw_sql"`
```

Comment:
Same comment as earlier – dangerous to expose `raw_sql` in the `Query` type where anyone with access can set it